services/src/main/java/org/exoplatform/officeonline/EditorService.java

```diff
@@ -10,6 +10,8 @@
 
 import org.apache.commons.io.input.AutoCloseInputStream;
```

Comment:
Why we need it?

Reply:
It's used in getContent(..) method which returns DocumentContent with InputStream of the file content. We need to be sure that the data stream will be closed when EoF will be reached.